ingest-app/src/cmr/ingest/api/routes.clj

```diff
@@ -74,6 +74,11 @@
            (acl/verify-ingest-management-permission ctx :update)
            (jobs/trigger-partial-refresh-collection-granule-aggregation-cache
             ctx)
+           {:status 200})
+     (POST "/trigger-granule-task-cleanup-job"
+           {ctx :request-context}
+           (acl/verify-ingest-management-permission ctx :update)
+           (jobs/trigger-bulk-granule-task-cleanup ctx)
```

Comment:
Is this a background job that runs periodically to clean up or is this job triggered by people invoking the endpoint?

Reply:
Both! In the jobs.clj file I have it set to run at a set interval (every 24 hours)